packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -718,6 +718,15 @@ export abstract class FluidDataStoreContext extends TypedEventEmitter<IFluidData
         this.channel.reSubmit(innerContents.type, innerContents.content, localOpMetadata);
     }
 
+    public rollback(contents: any, localOpMetadata: unknown) {
```

Comment:
Probably out of scope since I see precedent in `applyStashedOp`, but too bad this is `any` and gets casted :(  If it's something we would want to revisit reasonably soon, maybe open a backlog item to rework these APIs?

Reply:
I think the nature of messages requires a cast to happen somewhere, although we could do it higher up the call stack. Not sure of the relative priority of fixing this up.

Reply:
i don't think we can do anything at this layer. we could setup generics in shared object so the code there is a bit more structured